libs/core/src/lib/button/button.component.ts

```diff
@@ -37,14 +37,21 @@ import { applyCssClass } from '@fundamental-ngx/core/utils';
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
         '[attr.type]': 'type',
-        '[attr.disabled]': '_disabled || null'
+        '[attr.disabled]': '_disabled || null',
+        '[attr.aria-label]': 'ariaLabel ? ariaLabel : specialButtonType.includes(fdType) ? label !==null ?  label +" ,"+ fdType : glyph !== undefined || glyph !== null ? fdType +","+ glyph.split("-").join(" ")  : title : title'
     }
 })
 export class ButtonComponent extends BaseButton implements OnChanges, CssClassBuilder, OnInit,  OnDestroy {
     /** The property allows user to pass additional css classes. */
     @Input()
     class = '';
 
+    /** @hidden */
+    specialButtonType: Array<string> = ['emphasized', 'positive', 'negative', 'attention'];
```

Comment:
Could you please care about supporting i18n.

Reply:
this array is used to handle only button types that has to be read out by default., other button types are not required since it creates unwanted confusion this array.this is used to handle internally.

Reply:
Got it, so maybe we need to rename the field to avoid such misunderstanding in feature. I guess we also has alll the button types as enum, do we? In that case we can use as items for that array enum values from there.